### ISSUE TITLE
Nodecg-io Dashboard can't Login When in an OBS Custom Dock

### ISSUE THREAD

**Issue by devJimmyboy** [codeoverflow-org/nodecg-io#311]
### Description

Fullbleed panel for NodeCG-io doesn't allow logging in to the config when located in an OBS Browser Dock.

From the get-go, I'd say it has to do with functionality with OBS's Chromium Version. 
I'd make sure to build with a browserlist of `chrome >= 74` to account for OBS's older browser support. 
(although you could already be doing this, as I haven't checked nor know how to)

### How to reproduce

- Start NodeCG with Nodecg-io installed.
- Add https://localhost:PORT/dashboard to OBS Custom Docks
- Switch to Nodecg-io tab & Enter Password.
- Click Login
- ....nothing....
- Check console through chromium debug flag & find this:

### Error Log 
 
```
Uncaught (in promise) ReferenceError: monaco is not defined
    at showInMonaco (main.bundle.js:374)
    at onInstanceSelectChange (main.bundle.js:185)
    at selectServiceInstance (main.bundle.js:338)
    at renderInstances (main.bundle.js:319)
    at Config.<anonymous> (main.bundle.js:139)
    at Config.emit (main.bundle.js:8067)
    at updateDecryptedData (main.bundle.js:561)
    at main.bundle.js:519
    at Generator.next (<anonymous>)
    at fulfilled (main.bundle.js:960)
```

**Comment by daniel0611** [comment]
I've succesfully reproduced this bug with OBS Studio 27.1.3 (the currently latest version) and fixed it with #312.
You can either use the current development version for this fix or wait till (approximately) friday, as I'll release nodecg-io version `0.2.0` soon.

**Comment by devJimmyboy** [comment]
Awesome! Thanks a lot! Can't wait for 0.2.0!
